x/auth/ante/fee_test.go

```diff
@@ -59,7 +59,7 @@ func TestEnsureMempoolFees(t *testing.T) {
 	// msg and signatures
 	msg := testdata.NewTestMsg(accs[0].acc.GetAddress())
 	feeAmount := testdata.NewTestFeeAmount()
-	gasLimit := testdata.NewTestGasLimit()
+	gasLimit := uint64(15)
```

Comment:
Good idea, it makes the test easier to understand / read. In my version of the fee prioritization I'm scaling the gas based on fee amount (using `QuoRaw`)